zipkin-core/src/main/java/com/smoketurner/dropwizard/zipkin/HttpZipkinFactory.java

```diff
@@ -51,6 +56,22 @@ public void setBaseUrl(String baseUrl) {
         this.baseUrl = baseUrl;
     }
 
+    @Nullable
+    private Integer connectTimeout;
```

Comment:
Sorry, just seeing this PR now. Can we use Dropwizard’s Duration class in both of these so the use can control the TimeUnit. Otherwise, looks good.

Reply:
@jplock Wasn't aware of this option. Nice.